plugins/toolbox/README.md

```diff
@@ -0,0 +1,9 @@
+# toolbox
+
+This plugins adds `toolbox_prompt_info()` function. It shows the toolbox indicator ⬢ if you are running in a container and nothing if not.
```

Comment:
The first description line should avoid going into specifics. A better tagline would be "a prompt function for toolbox (insert link to project here)".